create/templates/shared/checkout_post_purchase/javascript.js.tpl

```diff
@@ -6,6 +6,20 @@
  *     completes
  */
 
+{{- if .Development.UsesNext }}
+import {
+  extend,
+  BlockStack,
+  Button,
+  Heading,
+  Image,
+  Layout,
+  TextBlock,
+  TextContainer,
+  CalloutBanner,
+  View,
+} from "@shopify/app/ui-extensions/post-purchase";
```

Comment:
Is there a reason to wrap the whole JS statement in a conditional? It's all the same stuff except this line, maybe we can just `if/else` this line to avoid duplication and the dangers therein?

Reply:
Good idea. I've updated the code to move the `if/else` to the module identifier.